### ISSUE TITLE
AJAX enabling/disabling: Accessibility: render changes to screen readers

### ISSUE THREAD

**Issue by audrasjb** [WordPress/wp-autoupdates#84]
[wp.a11y.speak](https://make.wordpress.org/accessibility/handbook/markup/wp-a11y-speak/) should be implemented to inform screen readers users that the action has been processed.

**Comment by ronalfy** [comment]
@audrasjb I put together a quick PR for this one in #86. I didn't spend too much time on it, but it does demonstrate what's possible.

**Comment by ronalfy** [comment]
Looks like we both did PRs for this. My apologies. I'll go ahead and close mine out.
